openslides_backend/models/fields.py

```diff
@@ -11,51 +11,62 @@ class Field:
     Base class for model fields.
     """
 
-    def __init__(self, description: str) -> None:
+    def __init__(self, description: str = "", **constraints: Any) -> None:
         self.description = description
+        self.constraints = constraints
 
     def get_schema(self) -> Schema:
         """
         Returns a JSON schema for this field.
         """
-        raise NotImplementedError
+        return dict(description=self.description, **self.constraints)
+
+    def extend_schema(self, schema: Schema, **kwargs: Any) -> Schema:
+        schema.update(kwargs)
+        return schema
 
 
-class IdField(Field):
+class IntegerField(Field):
     def get_schema(self) -> Schema:
-        return dict(description=self.description, type="integer", minimum=1,)
+        return self.extend_schema(super().get_schema(), type="integer")
 
 
-class IntegerField(Field):
+class PositiveIntegerField(IntegerField):
     def get_schema(self) -> Schema:
-        return dict(description=self.description, type="integer",)
+        return self.extend_schema(super().get_schema(), minimum=1)
 
 
-class PositiveIntegerField(Field):
+class IdField(PositiveIntegerField):
+    pass
+
+
+class TimestampField(Field):
```

Comment:
This should inherit from PositiveIntegerField

Reply:
Please also add a short comment or docstring what this integer is about (the UNIX timestamp I think).